helm/idol-nifi/templates/nifi/scripts-configmap.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nifi-ssl-cm
+  labels: {{- include "idol-nifi.labels" . | nindent 4 -}}
+    app: nifi
+data:
+  security.sh: |
```

Comment:
Easier to manage/edit if scripts are checked in separately and transcluded by `.Files.Get`?

Reply:
[8f12d06](https://github.com/opentext-idol/idol-containers-toolkit/pull/12/commits/8f12d06cfce23e46b00d89a6623ee25d4c4664c3)